pkg/cli/cmd/cli.go

```diff
@@ -1,44 +1,71 @@
 package cmd
 
 import (
-	"errors"
-)
+	"bufio"
+	"fmt"
+	"os"
+	"strings"
 
-var (
-	ErrNotRunning = errors.New("topaz is not running, use 'topaz start' or 'topaz run' to start")
-	ErrIsRunning  = errors.New("topaz is already running, use 'topaz stop' to stop")
-	ErrNotServing = errors.New("topaz gRPC endpoint not SERVING")
+	"github.com/aserto-dev/topaz/pkg/cli/cmd/authorizer"
+	"github.com/aserto-dev/topaz/pkg/cli/cmd/directory"
 )
 
-type FormatVersion int
+type SaveContext bool
 
 const (
-	V2 FormatVersion = 2
-	V3 FormatVersion = 3
+	CLIConfigurationFile = "cli_config.json"
```

Comment:
Suggest renaming this to `config.json` or `topaz.json`.
The fact it resides in XDG_CONFIG_HOME already reveals its nature. 
The name `cli_*` feels odd.